lua/trunk.lua

```diff
@@ -226,6 +257,30 @@ end
 
 -- Startup, including attaching autocmds
 local function start()
+	local cliVersion = getCliVersion()
+	logger.info("Running on CLI version:", cliVersion)
+	if cliVersion == nil then
+		logger.error("nil CLI version")
+		print(
+			"The Trunk Neovim extension requires Trunk CLI version >= 1.17.0 - we could not determine your Trunk CLI version"
+		)
+		print("The extension will not run until you upgrade your CLI version")
+		print("Please run `trunk upgrade cli` to upgrade your CLI version")
+		return
+	end
+	if not checkCliVersion("1.17.0") then
+		logger.error("Trunk CLI version must be >= 1.17.0")
+		print("The Trunk Neovim extension requires Trunk CLI version >= 1.17.0 - you currently have " .. cliVersion)
+		print("The extension will not run until you upgrade your CLI version")
+		print("Please run `trunk upgrade cli` to upgrade your CLI version")
+		return
+	end
+	if not checkCliVersion("1.17.2") then
+		logger.warn("Trunk CLI version should be >= 1.17.2")
+		print("The Trunk Neovim extension prefers Trunk CLI version >= 1.17.2 - you currently have " .. cliVersion)
+		print("The extension will still run, but there are known bugs with bazel")
+		print("Please run `trunk upgrade cli` to upgrade your CLI version")
```

Comment:
Change copy. If they want the bug details they can check the changelog.

```
Detected stale Trunk CLI version ${cliVersion}. Please run `trunk upgrade` to get the latest improvements and fixes for Neovim.
```
That way we can easily bump this whenever we need to for new fixes and we don't have to keep track of all the bugs we've fixed.